content/source/docs/cloud/registry/using.html.md

```diff
@@ -55,6 +55,16 @@ Click a module's "Details" button to view its details page. Use the "Versions" d
 
 ![Terraform Cloud screenshot: a module details page](./images/publish-module-details.png)
 
+### Viewing Nested Modules
+If a module contains nested modules following the [standard module structure](/docs/modules/structure.html), then a "Submodules" dropdown will appear below the module source information. 
```

Comment:
I have some questions! 

- It looks like the convention is that a nested module can be used independently if it contains a `README`/`README.md` file. Is that how the registry decides which submodules to let you browse? Or does it include other submodules as well? 
- Do nested modules get their own independent source addresses in the registry? As in, if I paste the snippet from the "provision instructions" block for a submodule, will it just work without me needing to also include the parent module in my configuration? 
    - If so, what's the structure of a source address for a nested module?

Reply:
there is no checking by the registry if you are using nested modules with or without a readme. They are just sorted differently in the dropdown:
![image](https://user-images.githubusercontent.com/4753589/104224258-98c5e400-53f9-11eb-8318-b664e3ba9712.png)

Basically it works the same as the public registry does today (which I couldn't find any documentation for the public registry)

Reply:
Also the source is different for submodules and examples:
```
module "project-factory_fabric-project" {
  source  = "tfe-zone-1a04f900.ngrok.io/example_corp/project-factory/google//modules/fabric-project"
  version = "9.2.0"
  # insert required variables here
}
```